<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Denis Khachko</author>
 <class>UmkiPult</class>
 <widget class="QWidget" name="UmkiPult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>254</width>
    <height>416</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>254</width>
    <height>416</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>148</red>
        <green>143</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>222</red>
        <green>215</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>185</red>
        <green>179</green>
        <blue>166</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>74</red>
        <green>71</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>99</red>
        <green>95</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>148</red>
        <green>143</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>201</red>
        <green>199</green>
        <blue>194</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>148</red>
        <green>143</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>222</red>
        <green>215</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>185</red>
        <green>179</green>
        <blue>166</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>74</red>
        <green>71</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>99</red>
        <green>95</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>148</red>
        <green>143</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>201</red>
        <green>199</green>
        <blue>194</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>74</red>
        <green>71</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>148</red>
        <green>143</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>222</red>
        <green>215</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>185</red>
        <green>179</green>
        <blue>166</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>74</red>
        <green>71</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>99</red>
        <green>95</green>
        <blue>89</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>74</red>
        <green>71</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>74</red>
        <green>71</green>
        <blue>66</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>148</red>
        <green>143</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>148</red>
        <green>143</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>148</red>
        <green>143</green>
        <blue>133</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Пульт</string>
  </property>
  <widget class="QPushButton" name="TempB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Temp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StenaB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Стена
Закр</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="UpB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RightB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LeftB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DownB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RadB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SvobodnoB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Свободно
Чисто</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="ClearLog">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/stop.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="CenterB">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/robo_field.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>61</width>
     <height>61</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="ToKumir">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Скопировать</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/stop.png</normalon>
    </iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
